plugin-server/src/worker/ingestion/team-manager.ts

```diff
@@ -106,18 +118,31 @@ export class TeamManager {
             ingested: team.ingested_event,
         })
 
-        await this.cacheEventNamesAndProperties(team.id)
+        await this.cacheEventNamesAndProperties(team.id, event)
```

Comment:
:thinking: Should we use `await Promise.all([ ... ])` here?

Reply:
I'm afraid this can substantially increase the number of open postgres connections in short bursts. Thus safer, for now, to keep this sequential.

The first function here should anyway run before all the others, but the `sync`-s could be done in parallel in the future, though, again, I'm a bit afraid of excess postgres parallelisation.

Reply:
Also, since this should only be run the first time a unique event or property is ever seen, we might just be prematurely optimising here.